pom.xml

```diff
@@ -85,6 +91,13 @@ jar, tar, zip, dump, 7z, arj.
       <version>${powermock.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.10</version>
+      <scope>test</scope>
+    </dependency>
+    
```

Comment:
I'd prefer encoding things by hand rather than adding a new dependency just for tests.